integration/pull_test.go

```diff
@@ -501,6 +504,148 @@ level = "debug"
 	}
 }
 
+// TestPullWithArbitraryBlobAsZtoc tests the snapshotter behavior if an arbitrary blob is passed
+// as a Ztoc. In this case, the flatbuffer deserialization will fail, which will lead
+// to the snapshotter mounting the layer as a normal overlayfs mount.
+func TestPullWithArbitraryBlobAsZtoc(t *testing.T) {
+	regConfig := newRegistryConfig()
+	getContainerdConfigYaml := func(disableVerification bool) []byte {
```

Comment:
Can we move `getContainerdConfigYaml` and `getSnapshotterConfigYaml` to a common place like `testutil`? Both are duplicated multiple times. (2 in `push_test.go`, 4 in `pull_test.go`)

https://github.com/search?q=repo%3Aawslabs%2Fsoci-snapshotter+%22getContainerdConfigYaml+%3A%3D%22&type=code

Reply:
I think it should be done as part of #247. Removing it will touch multiple tests, and that seems out of scope of this PR.

Reply:
SGTM